test-integration/samples/jdl-entities/entities.jdl

```diff
@@ -8,4 +8,11 @@ entity JdlFieldTest {
   value String
 }
 
-dto JdlFieldTest with mapstruct
+@ChangelogDate(20200804035353)
+entity MapstructExpressionRelatedFieldTest
+
+relationship OneToMany {
```

Comment:
Am I wrong or there's no mapstruct expression in this description?

Reply:
It's on line 7 ("value"). And the issue here is really about referencing such a column within the relationship :D